lib/cocoapods-core/specification/dsl/attribute.rb

```diff
@@ -31,7 +31,7 @@ def initialize(name, options)
 
           @multi_platform = options.delete(:multi_platform) { true       }
           @root_only      = options.delete(:root_only)      { false      }
-          @test_only      = options.delete(:test_only)      { false      }
+          @spec_types     = options.delete(:spec_types)     { [:app]     }
```

Comment:
default spec types should be all of them?

should also verify that `(@spec_types - Specification::VALID_TYPES).empty?`

(that constant might also need to be introduced)

Reply:
thanks that was my question above. will change